mmhuman3d/data/data_structures/human_data.py

```diff
@@ -610,6 +613,8 @@ def __get_slice_dim__(self) -> dict:
                             sub_value_len = len(value[sub_key])
                             if sub_value_len != self.__data_len__:
                                 ret_dict[key][sub_key] = None
+                            elif 'dim' in value:
```

Comment:
What about swtich dim and sub_value_len? If `dim` has been configured, it has the highest priority.

Reply:
Done!